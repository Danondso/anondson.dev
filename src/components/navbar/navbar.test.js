import React from 'react';
import renderer from 'react-test-renderer';
import NavBar from './navbar';

describe('navbar Test Suite', () => {
  it('renders correctly with required props', () => {
    const tree = renderer.create(<NavBar />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
