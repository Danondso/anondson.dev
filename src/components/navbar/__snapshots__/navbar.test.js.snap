// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navbar Test Suite renders correctly with required props 1`] = `
<section
  alt="Colored background containing contact and footer info"
  style={
    Object {
      "backgroundColor": "var(--background-yellow)",
      "boxShadow": "0 3px 2px rgba(0, 0, 0, 0.15)",
      "padding": "4rem",
    }
  }
>
  <nav
    style={
      Object {
        "float": "right",
        "marginTop": "-1rem",
      }
    }
  >
    <a
      href="/"
    >
      Home
    </a>
  </nav>
</section>
`;
